index.cjs

```diff
@@ -0,0 +1,66 @@
+import bookIDgenerator from './modules/books.js';
+import AddBook from './modules/addBook.js';
+
+import clearBooks from './modules/clearAllBooks.js';
+import printBooks from './modules/showBooks.js';
+import { DateTime } from './modules/luxon.js';
+
+const now = DateTime.now();
+document.getElementById('date').textContent = now;
```

Comment:
- [OPTIONAL] Great job using `Luxon` library 💯👌 I would like to give you a small tip to display the date in a nicer way I suggest using `toLocaleString(DateTime.DATETIME_MED)` method 😄
> For example 
```
const now = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
document.getElementById('date').textContent = now;
```

![image](https://user-images.githubusercontent.com/45231276/178454968-f206d1bf-fc30-4204-bcc2-0731bd1003d4.png)

Reply:
@Bondok6 Thanks so much for your review and your kind intentions to help..It somehow shows you love what you do! Nice to find peole like your around ..will take all your advices, have a lovely day!